vue/wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,162 @@
+<template lang='pug'>
+  .register-container
+    img.wolox-icon(alt='Vue logo' src='../assets/wolox_logo.svg')
+    form.form-container(@submit.prevent="onSubmit")
+      p.title-form
+        |{{labels.title}}
+      .input-text-container(v-for='(field, index) in fields')
+        label.input-text-label(:for='field.name')
+          |{{ field.label }}
+        input.input-text-content(:id='field.name' v-model='user[field.name]' )
+      button.base-form-button
+        |{{ labels.signUp }}
+    .container-button
+      button.base-form-button.login-button
+        |{{ labels.signIn }}
+</template>
+
+<script>
+
+const fieldsArray = [
+  {
+    name: 'fisrtName',
+    label: 'First name'
+  },
+  {
+    name: 'lastName',
+    label: 'Last name'
+  },
+  {
+    name: 'email',
+    label: 'Email'
+  },
+  {
+    name: 'password',
+    label: 'Password'
+  }
+]
+
+const labels = {
+  title: 'BOOKS',
+  signUp: 'Sign up',
+  signIn: 'Login'
+}
+
+export default {
+  name: 'register',
+  props: {
+    fields: {
+      type: Array,
+      default: () => fieldsArray
+    },
+    labels: {
+      type: Object,
+      default: () => labels
+    }
+  },
+  data () {
+    return {
+      user: {}
+    }
+  },
+  methods: {
+    onSubmit () {
+      console.log(this.user, 'user info')
+    }
+  }
+}
+</script>
+
+<style scoped lang='scss'>
+  @import 'src/scss/colors';
+  @import 'src/scss/fonts';
+
+  .register-container {
+    background-color: $wild-sand;
+    border-top: 5px solid $cerulean;
+    margin: 50px auto;
+    max-width: 400px;
+    width: 100%;
+  }
+
+  .wolox-icon {
+    margin: 35px 30px 10px;
+  }
+
+  .title-form {
+    color: $black;
+    font-size: $title-form;
+    font-weight: bold;
+    letter-spacing: 5px;
+    margin-bottom: 30px;
+    text-align: center;
+  }
+
+  .form-container {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    margin-bottom: 20px;
+    max-height: 600px;
+    width: 100%;
+  }
+
+  .input-text-container {
+    align-items: flex-start;
```

Comment:
This is the default value are you trying to override something?

Reply:
You are right 😄 .